web/libs/datamanager/src/components/Filters/FilterInput.jsx

```diff
@@ -11,13 +11,16 @@ export const FilterInput = ({ value, type, onChange, placeholder, schema, style
 
   return (
     <Input
-      rawClassName="h-full"
+      rawClassName="h-full font-size-12 h-8 min-h-8"
```

Comment:
what do these changes do?

Reply:
another mangled change to revert, my bad